app/src/main/java/com/github/se/gomeet/ui/mainscreens/profile/EditProfile.kt

```diff
@@ -79,22 +83,35 @@ fun EditProfile(
     nav: NavigationActions,
     userViewModel: UserViewModel = UserViewModel(UserRepository(Firebase.firestore))
 ) {
+
+  val countries = getCountries()
   val currentUser = remember { mutableStateOf<GoMeetUser?>(null) }
   val firstName = remember { mutableStateOf("") }
+  var firstNameValid by remember { mutableStateOf(false) }
   val lastName = remember { mutableStateOf("") }
-  val email = remember { mutableStateOf("") }
+  var lastNameValid by remember { mutableStateOf(false) }
+  var allUsers by remember { mutableStateOf<List<GoMeetUser>?>(null) }
+
   val username = remember { mutableStateOf("") }
+  var usernameValid by remember { mutableStateOf(false) }
+
   val phoneNumber = remember { mutableStateOf("") }
+  var phoneNumberValid by remember { mutableStateOf(false) }
+
   val country = remember { mutableStateOf("") }
+  var countryValid by remember { mutableStateOf(false) }
   val tags = remember { mutableStateOf(emptyList<String>()) }
   var isLoaded by remember { mutableStateOf(false) }
   val showPopup = remember { mutableStateOf(false) }
 
+  var firstClick by remember { mutableStateOf(true) }
+
   LaunchedEffect(Unit) {
-    currentUser.value = userViewModel.getUser(com.google.firebase.Firebase.auth.currentUser!!.uid)
+    allUsers = userViewModel.getAllUsers()
+    currentUser.value = allUsers!!.first { it.uid == Firebase.auth.currentUser!!.uid }
+    allUsers!!.minus(currentUser.value!!)
```

Comment:
This should be allUsers = allUsers!!.minus(currentUser.value!!), .minus() only returns a list without the specified element (it doesn't modify any lists)

Reply:
Indeed... This is fixed now